rails/db/migrate/20240726071341_create_active_storage_tables.active_storage.rb

```
@@ -0,0 +1,57 @@
+    end
+
+    create_table :active_storage_variant_records, id: primary_key_type do |t|
+      t.belongs_to :blob, null: false, index: false, type: foreign_key_type
+      t.string :variation_digest, null: false
+
+      t.index [ :blob_id, :variation_digest ], name: :index_active_storage_variant_records_uniqueness, unique: true
+      t.foreign_key :active_storage_blobs, column: :blob_id
+    end
+  end
+
+  private
```

rubocop

[convention] Keep a blank line before and after `private`.

(Layout/EmptyLinesAroundAccessModifier)